schema.graphql

```diff
@@ -365,60 +371,15 @@ type ALMDeployFactory_StrategyCreated_AmmPosition {
   logIndex: Int!
 }
 
-type ALMLPWrapper_Deposit {
-  id: ID!
-  sender: String!
-  recipient: String!
-  pool: String!
-  amount0: BigInt! @config(precision: 76)
-  amount1: BigInt! @config(precision: 76)
-  lpAmount: BigInt! @config(precision: 76)
-  totalSupply: BigInt! @config(precision: 76)
-  timestamp: Timestamp!
-  transactionHash: String!
+# ALM entity on a pool basis
+type ALM_LP_Wrapper {
+  id: ID! # {ALMaddress}_{chainId}
   chainId: Int!
-  blockNumber: Int!
-  logIndex: Int!
-}
-
-type ALMLPWrapper_Withdraw {
-  id: ID!
-  sender: String!
-  recipient: String!
   pool: String!
   amount0: BigInt! @config(precision: 76)
   amount1: BigInt! @config(precision: 76)
-  lpAmount: BigInt! @config(precision: 76)
-  totalSupply: BigInt! @config(precision: 76)
-  timestamp: Timestamp!
-  transactionHash: String!
-  chainId: Int!
-  blockNumber: Int!
-  logIndex: Int!
-}
-
-type ALMLPWrapper_Transfer {
-  id: ID!
-  from: String!
-  to: String!
-  value: BigInt! @config(precision: 76)
-  timestamp: Timestamp!
-  transactionHash: String!
-  chainId: Int!
-  blockNumber: Int!
-  logIndex: Int!
-}
```

Comment:
worthwhile checking with Dobri if he is using any of these event-based entities for ALM section on the monitoring dashboard